docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc

```diff
@@ -847,102 +884,107 @@ publish
   --connectTimeout  Int (default 5000) 
   --awaitTimeout    Int (default 120000)
   --stagingRelease  Boolean (default true)
-```
+----
 
 You also need to specify `release` as `true` or `false`, depending on whether you just want to stage your module
 on `oss.sonatype.org` or you want Mill to complete the release process to Maven Central.
 
 If you are publishing multiple artifacts, you can also use `mill mill.scalalib.PublishModule/publishAll` as described
-[here](https://com-lihaoyi.github.io/mill/page/common-project-layouts.html#publishing)
+https://com-lihaoyi.github.io/mill/page/common-project-layouts.html#publishing[here]
 
-## Structure of the `out/` folder
+== Structure of the `out/` folder
 
-The `out/` folder contains all the generated files & metadata for your build. It is structured with one folder
+The `out/` folder contains all the generated files &amp; metadata for your build. It is structured with one folder
 per `Target`/`Command`, that is run, e.g.:
 
-- `out/core/compile/`
-- `out/main/test/compile/`
-- `out/main/test/forkTest/`
-- `out/scalalib/compile/`
+* `out/core/compile/`
+* `out/main/test/compile/`
+* `out/main/test/forkTest/`
+* `out/scalalib/compile/`
 
 There are also top-level build-related files in the `out/` folder, prefixed as
 `mill-*`. The most useful is `mill-profile.json`, which logs the tasks run and time taken for the last Mill command you
 executed. This is very useful if you want to find out exactly what tasks are being run and Mill is being slow.
 
 Each folder currently contains the following files:
 
-- `dest/`: a path for the `Task` to use either as a scratch space, or to place generated files that are returned
-  using `PathRef`s. `Task`s should only output files within their given `dest/` folder (available as `T.dest`) to avoid
-  conflicting with other `Task`s, but files within `dest/` can be named arbitrarily.
+* `dest/`: a path for the `Task` to use either as a scratch space, or to place generated files that are returned
+ using `PathRef`s. `Task`s should only output files within their given `dest/` folder (available as `T.dest`) to avoid
```

Comment:
Not sure why, but this looks weird: https://lefou.github.io/mill/mill/Intro_to_Mill.html#_structure_of_the_out_folder
Whole "PathRef`s. `Task`s should only output files within their given `dest" is highlighted as code